subprojects/tooling-api/src/main/java/org/gradle/tooling/ProjectConnection.java

```diff
@@ -155,6 +156,13 @@
      */
     BuildActionExecuter.Builder action();
 
+    /**
+     * Notifies all daemons about the some changes made by an external tool, like an IDE.
```

Comment:
Superfluous `the` (and potentially `some`). I think the point is that an external process made the changes, doesn't have to be a tool (whatever that means).

Reply:
Fixed.